api/dbConfig/dev_seeds.sql

```diff
@@ -1,2 +1,15 @@
-INSERT INTO users (username, password) 
-VALUES ('admin', 'admin');
\ No newline at end of file
+INSERT INTO users (username, password, habit_id)
+VALUES
+('trumpetgems', 'secret123', 'habit1' ),
+('sparrow', 'secret789', 'habit2' )
+
+
+
+INSERT INTO habit_type (habit_id, user, frequency, completed_today, num_times_comp, last_comp)
+VALUES
+('habit1', 'coding', 'everyday', 'true', 5, 2020-10-05 )
```

Comment:
if we have the comp_dates num_times completed is not needed. 
also make the habit id in the schema serial and as a primary key, so it is auto generated.
Frequency probably best to define as as integer so everyday would be 1, every two days 2, once a week 7